custom.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module "*.svg" {
+  import React = require("react"); // eslint-disable-line no-unused-vars
+  export const ReactComponent: React.SFC<React.SVGProps<SVGSVGElement>>;
+  const src: string;
+  export default src;
+}
```

Comment:
This is so that TypeScript accepts SVG files can be imported as either a src string or a react component